src/naive/date/mod.rs

```diff
@@ -1201,52 +1201,46 @@ impl NaiveDate {
     ///
     /// # Errors
     ///
-    /// Returns `None` if:
-    /// - The resulting date does not exist (February 29 in a non-leap year).
-    /// - The year is out of range for a `NaiveDate`.
+    /// - Returns [`Error::DoesNotExist`] when the resulting date does not exist.
```

Comment:
@djc suggested at some point putting 'This method returns:' above the list, and removing 'Returns' from every line. I'm trying to stick to that style but am not consistent yet.

Reply:
Would prefer that over repeating it.

Reply:
Done